### ISSUE TITLE
FRRouting (FRR) stand-alone version and batfish

### ISSUE THREAD

**Issue by Viktort-rf** [batfish/batfish#8020]
Hello!
Can batfish work with an "independent" FRR? Not paired with cumulus linux

**Comment by ratulm** [comment]
@Viktort-rf -Batfish does FRR in the context of SONiC and Cumulus Linux at the moment. 

Your standalone FRR is just a software router?  If so, options:
 - Fool Batfish into thinking that it is being used in the context of Cumulus, e.g., by adding empty files for ports and interfaces.
 - Hack Batfish.

**Comment by Viktort-rf** [comment]
> @Viktort-rf -Batfish does FRR in the context of SONiC and Cumulus Linux at the moment.
> 
> Your standalone FRR is just a software router? If so, options:
> 
> * Fool Batfish into thinking that it is being used in the context of Cumulus, e.g., by adding empty files for ports and interfaces.
> * Hack Batfish.

Yes, I have a soft router based on a Linux VM with a FRR package.
I made concatenated files for "cumulus". And in bfq.nodeProperties in Configuration_Format I see CUMULUS_CONCATENATED. But any configuration analysis does not work (in this example, basic ospf). Output - empty fields :( 
Examples in the attachment.
[ngw22.txt](https://github.com/batfish/batfish/files/8038862/ngw22.txt)
[questions_batfish.txt](https://github.com/batfish/batfish/files/8038863/questions_batfish.txt)
[ural-noc1.txt](https://github.com/batfish/batfish/files/8038864/ural-noc1.txt)
[batfish_out.xlsx](https://github.com/batfish/batfish/files/8038865/batfish_out.xlsx)

**Comment by ratulm** [comment]
What does bf.q.initIssues() show? 

It may be easier to iterate via Slack: https://join.slack.com/t/batfish-org/shared_invite/enQtMzA0Nzg2OTAzNzQ1LTcyYzY3M2Q0NWUyYTRhYjdlM2IzYzRhZGU1NWFlNGU2MzlhNDY3OTJmMDIyMjQzYmRlNjhkMTRjNWIwNTUwNTQ

**Comment by Viktort-rf** [comment]
@ratulm thanks for the link to slack :)

It turned out to collect information about ospf sessions! :)
The problem was the syntax config. Standalone FRR (ver 8.1 git) has two types of ospf configuration, via "network" and via "interfaces".
In my first configurations, it was done through interfaces.
And after I executed bfq.initIssues, I saw error parsing and changed the ospf configuration to "network" and everything worked. 
But it still turns out that there is no native support for standalone FRR :((
